qemu/tests/usb_host.py

```diff
@@ -84,12 +84,6 @@ def usb_dev_unplug():
     match_add += "idVendor=%s, idProduct=%s" % (vendorid, productid)
     match_del = "USB disconnect"
 
-    error_context.context("Check usb device [%s] on host" % usb_options, logging.info)
-    try:
-        process.system(lsusb_cmd, shell=True)
-    except:
-        test.cancel("Device %s not present on host" % device)
```

Comment:
The first commit is depends on the vt patch https://github.com/avocado-framework/avocado-vt/pull/2617, which already check with lsusb.   So the second commit  "move host usb device checking to avocado-vt " can be merged to the first commit directly.

Reply:
Updated, thanks!